utils_cv/detection/keypoint.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import Dict, List
+import numpy as np
+import torch
+
+
+class COCOKeypoints:
+    """ Keypoints class for COCO datasets.
+
+    The interpretations of different categories of COCO Keypoints are the
+    same.  The only difference is the labels and skeleton (connection order)
+    for various categories.
+    """
+
+    def __init__(
+        self,
+        keypoints: np.ndarray,
+        meta: Dict[str, List] = None,
+    ):
+        """
+        Args:
+            keypoints:
+            meta:
+        """
+        if meta is not None:
+            self._meta = meta
+
+        self.keypoints = keypoints
+        if self.keypoints is not None:
+            self._init_keypoints()
+
+    def _init_keypoints(self):
+        self.keypoints = np.asarray(self.keypoints, dtype=np.float)
+        if self.keypoints.ndim != 3:
+            self.keypoints = self.keypoints.reshape(
+                (-1, len(self.meta["labels"]), 3)
+            )
+        assert self.is_valid()
+        self.standardize()
+
+    @property
+    def meta(self):
```

Comment:
Can we remove these getter and setter methods, since can just call COCOKeypoints.meta instead

Reply:
Done